### PR TITLE
NEPT-18: Move all subsites to NEXTEUROPA_METATAGS.

### PR DESCRIPTION
**Pull request by drishu**
### Description

Migrating multisite_metatags to nexteuropa_metatags. Enable nexteuropa_metatags by default. Uninstall multisite_metatags.

### Change log

- Added:
  - Hook to enable nexteuropa_metatags, migrate meta data and disable multisite_metatags.
  - README.md.
  - Variables nexteuropa_classification, nexteuropa_og_image, nexteuropa_creator, nexteuropa_reference.
  - File token.
  - nexteuropa_metatags.metatag.inc.
- Removed:
  - cce_basic_config_form_system_site_information_settings_alter().
  - nexteuropa_metatags.features.metatag.inc.

### CODE REVIEW

File: profiles/common/modules/features/nexteuropa_metatags/nexteuropa_metatags.metatag.inc [hunk 1]
```diff
@@ -0,0 +1,290 @@
...
+  );
+
+  $options = array();
+  if ($vocabulary = taxonomy_vocabulary_machine_name_load('metatags_ess_gf')) {
+    $terms = taxonomy_get_tree($vocabulary->vid);
+    foreach ($terms as $term) {
+      $term_full = taxonomy_term_load($term->tid);
+      $options[$term_full->field_meta_value[LANGUAGE_NONE][0]['value']] = str_repeat('-', $term->depth) . $term->name;
+    }
+  }
```
Comment (Fefaine) [inline profiles/common/modules/features/nexteuropa_metatags/nexteuropa_metatags.metatag.inc:248]: You are closing the condition too soon, we should only create the info tags ess gf entries if there is a taxonomy metatags_ess_gf

File: profiles/common/modules/features/nexteuropa_metatags/nexteuropa_metatags.metatag.inc [hunk 2]
```diff
@@ -0,0 +1,290 @@
...
+  $info['tags']['ESS_RF'] = array(
+    'label' => t('ESS RF'),
+    'description' => t('Restricted search'),
+    'class' => 'DrupalTextMetaTag',
+    'group' => 'ESS_metatags',
+  );
+
+  // Default values for each nexteuropa meta tag.
+  $ne_defaults = array(
+    'description' => '',
```
Comment (Fefaine) [inline profiles/common/modules/features/nexteuropa_metatags/nexteuropa_metatags.metatag.inc]: we are adding a description here, but we add it again afterwards

File: profiles/common/modules/features/nexteuropa_metatags/nexteuropa_metatags.metatag.inc [hunk 3]
```diff
@@ -0,0 +1,290 @@
...
+    'group' => 'nexteuropa',
+    'element' => array(
+      '#theme' => 'metatag_property',
+    ),
+  );
+
+  $info['tags']['creator'] = array(
+    'label' => t('Creator'),
+    'description' => t('Ex. COMM/DG/UNIT (max. 30 characters).<br/>This text will supply information to METADATA "Creator".<br/>See the <a href="http://ec.europa.eu/ipg/content/optimise/metadata/annex1_en.htm#section_2" target="_blank">IPG guidelines</a> for more information.'),
+    'context' => array('global'),
```
Comment (Fefaine) [inline profiles/common/modules/features/nexteuropa_metatags/nexteuropa_metatags.metatag.inc]: This should move to $ne_defaults

**Comment by Fefaine** [comment]
I added some comments in the PR file.
I think we should also upgrade the metatags module.
I have this error after i ran the hook_update
Warning: Invalid argument supplied for foreach() in views_theme() (line 154 of /home/ec2-user/environment/platform-dev/profiles/multisite_drupal_standard/modules/contrib/views/views.module).

I could not identify where is comes from

**Review by Fefaine — CHANGES_REQUESTED** [review]

**Review by AlbaParrilla — DISMISSED** [review]

**Review by Fefaine — DISMISSED** [review]

**Review by AlbaParrilla — APPROVED** [review]
Ok for me

**Review by tomv-be — APPROVED** [review]

**Pull request merged**